src/main/resources/hudson/plugins/ws_cleanup/WsCleanup/config.jelly

```diff
@@ -45,5 +45,8 @@ DEALINGS IN THE SOFTWARE. -->
         <f:entry title="${%External Deletion Command}" help="/plugin/ws-cleanup/help/externalDelete.html">
             <f:textbox field="externalDelete" />
         </f:entry>
+        <f:entry title="${%Clean asynchronously}" help="/plugin/ws-cleanup/help/cleanAsynchronously.html">
+            <f:checkbox field="cleanAsynchronously" checked="${it.cleanAsynchronously}" />
```

Comment:
Why not call this "Fail the build if cleanup fails"? I doubt anyone would particularly care about sync/async if it weren't for that bug. Right now this looks like one of the options you look back on two years later, wondering why there are all of these useless checkboxes.